platform/core/src/services/MeasurementService/MeasurementService.ts

```diff
@@ -517,6 +517,7 @@ class MeasurementService extends PubSubService {
     }
 
     let measurement = {};
+    if (annotationType === 'Crosshairs') return;
```

Comment:
Could you just add a mapping for crosshairs, even if it doesn't do anything?  There are other tools like probe which could also get empty mappings to suppress errors.

Reply:
That makes sense, I'll have Crosshairs pretty much copy the mapping for Probe tool

Reply:
This has caused an unexpected behaviour in OHIF recently when you open MPR and you open Crosshairs since it has mapping it prompts for tracking. I will remove this in a PR 

![image](https://user-images.githubusercontent.com/7490180/233080167-7ff2c04f-ff94-4420-a4f1-5247dd640210.png)